### PR TITLE
spike/SC-1517

### PR DESCRIPTION
**Pull request by abnepc**
## Jira Story
[SC-1517](https://epcpower.atlassian.net/browse/SC-1517)
## About
Investigate/remove `sunspecdemo` from the repo
## Associated Pull Requests
* [stlib](https://github.com/epcpower/stlib/pull/48)
* [pm](https://github.com/epcpower/pm/pull/93)
* [grid-tied](https://github.com/epcpower/grid-tied/pull/683)
* [rm-dc](https://github.com/epcpower/rm-dc/pull/21)
* [m-pm](https://github.com/epcpower/m-pm/pull/4)
* [m-bcu](https://github.com/epcpower/m-bcu/pull/7)
* [m-tcu](https://github.com/epcpower/m-tcu/pull/53)
## Reproduction Steps
N/A
## Validation
Delete your `.venv` folder and do a `poetry install`. Then make sure that `sunspecdemo` was not installed. 

[SC-1517]: https://epcpower.atlassian.net/browse/SC-1517?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+  "conventionalCommits.scopes": ["SC-1517"]
```
[the remark this answers is not in the document]
Comment (abnepc) [reply .vscode/settings.json]: [1bb6251](https://github.com/epcpower/st/pull/49/commits/1bb625123da0e3109112a7c584edcbd30d626f18) & [ee7f81e](https://github.com/epcpower/st/pull/49/commits/ee7f81e3f7d0d51d0e7ecdfa280c7da65b50f790)

File: pyproject.toml [hunk 2]
```diff
@@ -99,4 +94,4 @@ requires = [
     "poetry-core>=1.0.0",
     "poetry-dynamic-versioning",
 ]
-build-backend = "poetry.core.masonry.api"
+build-backend = "poetry.core.masonry.api"
```
[the remark this answers is not in the document]
Comment (abnepc) [reply pyproject.toml]: [58eb2fc](https://github.com/epcpower/st/pull/49/commits/58eb2fc8fe3d1decab7b1622223d945cdc3fb0ae)

File: pyproject.toml [hunk 3]
```diff
@@ -67,6 +61,7 @@ poetry-dynamic-versioning = "^0.13.1"
 black = "20.8b1"
 gitignoreio = "0.1.5"
 pyinstaller = "~=4.5"
+pyinstaller-hooks-contrib = "2022.10"
```
Comment (gordon-epc) [inline pyproject.toml:64]:
```suggestion
pyinstaller-hooks-contrib = "2022.10"  # Locks the version to avoid updating during `poetry update`
```

**Review by gordon-epc — CHANGES_REQUESTED** [review]

File: pyproject.toml (hunk 3 above)
Comment (abnepc) [reply pyproject.toml]: [1f48722](https://github.com/epcpower/st/pull/49/commits/1f48722e35ba51a422748e703d35da42e25b3c46)

**Review by gordon-epc — APPROVED** [review]

**Pull request merged**